x-pack/plugins/security_solution/public/common/containers/sourcerer/index.tsx

```diff
@@ -445,6 +445,7 @@ export const useSourcererDataView = (
       selectedPatterns,
       // if we have to do an update to data view, tell us which patterns are active
       ...(legacyPatterns.length > 0 ? { activePatterns: sourcererDataView.patternList } : {}),
+      sourcererDataView,
```

Comment:
Could we delete some redundant properties of this object? Like `dataViewId `, `browserFields` and `runtimeMappings `?